document-store/src/main/java/org/hypertrace/core/documentstore/GroupBy.java

```diff
@@ -0,0 +1,35 @@
+package org.hypertrace.core.documentstore;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.apache.commons.lang3.StringUtils;
+import org.hypertrace.core.documentstore.expression.Expression;
+
+@Data
```

Comment:
Can this be a value (and the constructor is implied)?

Reply:
I didn't know this annotation before. Thanks for the tip. :)